SingularityUI/app/components/requestDetail/RequestDetailPage.jsx

```diff
@@ -91,7 +91,9 @@ const mapDispatchToProps = (dispatch, ownProps) => {
     FetchRequest.trigger(ownProps.params.requestId, true),
     FetchActiveTasksForRequest.trigger(ownProps.params.requestId),
     FetchScheduledTasksForRequest.trigger(ownProps.params.requestId),
-    FetchTaskCleanups.trigger()
+    FetchTaskCleanups.trigger(),
+    FetchDeploysForRequest.trigger(ownProps.params.requestId, 5, 1),
```

Comment:
We don't want to be fetching these two here. These are actions that will auto-refresh on an interval. Since deploy history and request history are paginated, fetching this would reset the user back to the first page on a schedule. We should instead use the `then` values of the buttons you are editing to make sure the correct api calls are made in reaction to a user action.

Additionally, the deploy history won't be changing in our request deletion updates, only the request history